blocks/init/src/Blocks/components/share/assets/share.js

```diff
@@ -5,26 +5,26 @@ export class Share {
 
 	init() {
 		[...this.shareLinks].forEach((shareLink) => {
-			shareLink.addEventListener('click', (event) => {
+			shareLink.addEventListener('click', async (event) => {
 				event.preventDefault();
 
-				this.openShareDialog(event.currentTarget);
+				await this.openShareDialog(event.currentTarget);
```

Comment:
It's a little weird that opening a modal should be async but ok 😄  (I assume because `navigator.share ` runs asyncronously?)

Reply:
Correct, I got an error because of the navigator API (which is super experimental from what I've read) 😬